Cargo.toml

```diff
@@ -71,7 +72,7 @@ version = "1.0"
 
 [dev-dependencies]
 # async-std = "1.10.0"
-criterion = "0.3.5"
+criterion = "0.3.5"     # Remember to update audit.toml
```

Comment:
I'm not 100% sure what this comment means. Does this reference the change that's in this PR or is this a reminder to a future maintainer that they need to do something important? If it's the second, is there a link with more details that we can leave for future persons?

Reply:
the audit failure that is whitelisted is brought into the dependency tree via criterion. When we get a dependabot PR updating criterion we need to go and update audit.toml.

Reply:
Ah! Yes, that makes sense!